### PR TITLE
fix the error of the read of /INISH/FAIL and STRS_F/GLOBAL

### PR DESCRIPTION
**Pull request by SaidMamouri**
#### Description of the feature or the bug

Core dump because the array is not enough to save variable and error message due the non compatibily of the number of integration point in the tickness.
#### Description of the changes

Fix the issue by correcting the Id of the loop of the save and to save the right integration point .

### CODE REVIEW

File: starter/source/elements/initia/hm_read_inistate_d00.F [hunk 1]
```diff
@@ -2250,7 +2250,7 @@ SUBROUTINE HM_READ_INISTATE_D00 (
                         SIZE = NPG
                         CALL HM_GET_FLOAT_ARRAY('sigma_X'  ,TMPVAL1,SIZE,J,IS_AVAILABLE,LSUBMODEL,UNITAB)
                         CALL HM_GET_FLOAT_ARRAY('sigma_Y'  ,TMPVAL2,SIZE,J,IS_AVAILABLE,LSUBMODEL,UNITAB)
-                        CALL HM_GET_FLOAT_ARRAY('sigma_Z'  ,TMPVAL3,SIZE,J,IS_AVAILABLE,LSUBMODEL,UNITAB)
+                       !! CALL HM_GET_FLOAT_ARRAY('sigma_Z'  ,TMPVAL3,SIZE,J,IS_AVAILABLE,LSUBMODEL,UNITAB)
```
Comment (sebastienVilleneuve) [inline starter/source/elements/initia/hm_read_inistate_d00.F:2253]: Hi @SaidMamouri 

You're not reading this TMPVAL3 value anymore , but it is still used to fill SIGSH array line 2270.
Can you check ?

Thanks,
Sébastien

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
Hi Said,

Some check or change to perform on your side  before approving this change

Thanks,
Sébastien

File: starter/source/elements/initia/hm_read_inistate_d00.F (hunk 1 above)
Comment (sebastienVilleneuve) [inline starter/source/elements/initia/hm_read_inistate_d00.F:2253]: Hi  @SaidMamouri said you don't fill TMPVAL3 value anymore , but it is used after , line 2270

Sébastien

Comment (SaidMamouri) [reply starter/source/elements/initia/hm_read_inistate_d00.F]: Thx Seb.

**Review by sebastienVilleneuve — APPROVED** [review]

File: starter/source/elements/initia/hm_read_inistate_d00.F (hunk 1 above)
Comment (sebastienVilleneuve) [reply starter/source/elements/initia/hm_read_inistate_d00.F]: You're welcome , it is now ok , Thanks a lot @SaidMamouri

**Pull request merged**